rpc/core/status.go

```diff
@@ -4,12 +4,14 @@ import (
 	"bytes"
 	"time"
 
+	cmn "github.com/tendermint/tendermint/libs/common"
 	ctypes "github.com/tendermint/tendermint/rpc/core/types"
 	sm "github.com/tendermint/tendermint/state"
 	"github.com/tendermint/tendermint/types"
-	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+const consensusTimeout = time.Second
```

Comment:
I'd prefer this const be inside the method (closer to the usage). Is there a specific reason to keep it in the top of the file?

Reply:
Gonna move.

Reply:
But I don't see much valuein having the constants inside of methods.

Reply:
developer does not have to jump to definition in order to see the value